### PR TITLE
fix codebase style with php-cs-fixer

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: app/config/config.iniBACKUP [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[config]
```
Comment (flodolo) [inline app/config/config.iniBACKUP]: Local extra file.

File: app/config/config.iniDEV [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[config]
```
Comment (flodolo) [inline app/config/config.iniDEV]: Local extra file.

File: app/config/config.iniPROD [hunk 3]
```diff
@@ -0,0 +1,33 @@
+[config]
```
Comment (flodolo) [inline app/config/config.iniPROD]: Local extra file.

**Comment by pascalchevrel** [comment]
:/ stupid me, cleaning up my local files...

**Comment by flodolo** [comment]
Besides the extra files, it looks great. It even puts elements in alphabetical order *_*

**Pull request merged**